.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    branches:
```

Comment:
[Apparently](https://github.community/t/run-workflow-on-push-tag-on-specific-branch/17519/2), this configuration will run on pushes to master and on pushes to any tags starting with "v". The workaround seems to be removing the branch condition here and adding it as a job condition.

Reply:
Great catch!